### DISCUSSION TITLE
Nested Resources in tabs

### DISCUSSION THREAD

**Discussion by dejury** [filamentphp/filament#2975] [Help]
I have this application which has two seperate "models". 
For the sake of a good UI, I would love to create "one" resource, containing both "models".

So when I click on edit, I would have 2 tabs. Both tabs contain data from a different model.
Is this even possible?

**Comment by danharrin** [comment]
Not possible with resource, but you could do this with a custom page & the form/table builder with a little Livewire knowledge.
